libcxx/test/std/depr/depr.c.headers/stddef_h.compile.pass.cpp

```diff
@@ -42,11 +42,13 @@ static_assert(std::is_integral<ptrdiff_t>::value, "");
 static_assert((std::is_same<decltype(nullptr), nullptr_t>::value), "");
 static_assert(sizeof(nullptr_t) == sizeof(void*), "");
 #if TEST_STD_VER >= 11
-#  if TEST_STD_VER >= 20
-// P0767
-static_assert(std::is_trivial<max_align_t>::value, "");
+static_assert(std::is_trivially_copyable<max_align_t>::value, "");
+static_assert(std::is_trivially_default_constructible<max_align_t>::value, "");
 static_assert(std::is_standard_layout<max_align_t>::value, "");
-#  else
+#  if TEST_STD_VER < 26 // P3247R2
+static_assert(std::is_trivial<max_align_t>::value, "");
+#  endif
```

Comment:
I don't think this is necessary. We're testing `max_align_t`, not the type traits.

Reply:
Yeah. But some existing uses of `is_pod` look weird. I think it's better to remove them, otherwise some test files will become weirder.